src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -879,6 +881,19 @@ export default class CDPipeline extends Component<CDPipelineProps, CDPipelineSta
             key === 'preStage'
                 ? this.state.pipelineConfig.runPreStageInEnv
                 : this.state.pipelineConfig.runPostStageInEnv
+        function getOptionLabel(option:any) {
+            if (option.type === 'configmaps') {
+                return (<div style={{ display: 'flex', alignItems: 'center', marginLeft: '0px'}}>
```

Comment:
please move all the inline CSS in the function to scss file & try to use classes

Reply:
done